packages/components/src/components/scroller/scroller.examples.md

```diff
@@ -175,10 +181,39 @@ export default class Code extends Vue {
   </p-scroller>
 </div>`;
   }
+  scrollbar = `<div style="max-width: 600px">
+  <p-scroller hasScrollbar="true">
+    <p-text-list>
+      <p-text-list-item>
+        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+        dolore magna aliquyam erat, sed diam voluptua. <br />At vero eos et accusam et justo duo dolores et ea rebum.
+      </p-text-list-item>
+      <p-text-list-item>
+        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+        dolore magna aliquyam erat, sed diam voluptua. <br />At vero eos et accusam et justo duo dolores et ea rebum.
+        <p-text-list>
+          <p-text-list-item>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+            dolore magna aliquyam erat, sed diam voluptua. <br />At vero eos et accusam et justo duo dolores et ea
+            rebum.
+          </p-text-list-item>
+          <p-text-list-item>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+            dolore magna aliquyam erat, sed diam voluptua. <br />At vero eos et accusam et justo duo dolores et ea
+            rebum.
+          </p-text-list-item>
+        </p-text-list>
+      </p-text-list-item>
+    </p-text-list>
+  </p-scroller>`;
 }
 </script>
 
 <style scoped lang="scss">
+
+  :deep(p-scroller) {
+    white-space: nowrap;
```

Comment:
why is this necessary?

Reply:
@stephanschroeter It's needed for the scrollbar example:

<img width="1002" alt="Bildschirm­foto 2023-03-17 um 13 57 03" src="https://user-images.githubusercontent.com/2981884/225912315-f403f017-9134-4aa4-b03d-7155a69f20ea.png">

<img width="1048" alt="Bildschirm­foto 2023-03-17 um 13 56 52" src="https://user-images.githubusercontent.com/2981884/225912310-b0995d85-d26b-47e3-939f-13a018cf769f.png">

I changed it to `:deep(p-scroller > p-text-list)` to be more specific.